packages/atomic/cypress/integration/facet-numeric.cypress.ts

```diff
@@ -0,0 +1,221 @@
+import {setUpPage, shouldRenderErrorComponent} from '../utils/setupComponent';
```

Comment:
Similar comment to the dateFacet, naming this file `numeric-facet.cypress.ts` unless there is a reason not to,